scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -1,10 +1,43 @@
 #!/usr/bin/env bash
 set -ex
 
-if [[ $AMI_TYPE != "al2gpu" && $AMI_TYPE != "al2keplergpu" ]]; then
+if [[ $AMI_TYPE != "al2gpu" && $AMI_TYPE != "al2keplergpu" && AMI_TYPE != "opengpu" ]]; then
     exit 0
 fi
 
+if [[ $AMI_TYPE == "nvidiaopen" ]]; then
+    sudo yum install -y yum-plugin-versionlock \
+        yum-utils
+    sudo amazon-linux-extras install epel -y
+
+    # disable amzn2 in favor of rh repo
+    sudo yum-config-manager --disable amzn2-nvidia
+    sudo yum-config-manager --add-repo=https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/cuda-rhel7.repo
+    sudo yum-config-manager --enable cuda-rhel7.repo
+
+    # install open dkms from rh repo
+    sudo yum install -y kmod-nvidia-open-dkms \
```

Comment:
need to specify version to match al nvidia version
sudo yum install -y nvidia-kmod-common-535.54.03
sudo yum install -y kmod-nvidia-open-dkms-535.54.03